components/eam/src/physics/cam/cloud_diagnostics.F90

```diff
@@ -181,6 +186,7 @@ subroutine cloud_diagnostics_init()
 
 
     if (history_amwg) then
+       call add_default ('CLOUD   ', 1, ' ')
```

Comment:
Let's make our new TOT_CLD_FRAC the default output rather than CLOUD...

Reply:
Though I guess that would give bad output if we ever didn't use SHOC... And we never do history_amwg for high-res SCREAM runs. So never mind. Don't bother with this change.